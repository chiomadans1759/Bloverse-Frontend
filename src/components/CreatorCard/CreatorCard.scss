#bl-author-card {
  position: fixed;
  right: 2.5rem;
  bottom: 1.5rem;
  width: 12.125rem;
  .author-image {
    width: 2.5rem;
    height: 2.5rem;
    object-fit: cover;
  }
  .author-name {
    font-size: 1rem;
    color: #ffffff;
  }
  .message {
    font-size: 0.875rem;
    color: rbga(255, 255, 255, 0.65);
  }
}
