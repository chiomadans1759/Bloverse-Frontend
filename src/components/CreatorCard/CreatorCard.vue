<template>
  <main id="bl-author-card">
    <div class="d-none d-md-block d-lg-block">
      <div class="row">
        <div class="col-md-2 align-self-center p-0">
          <img class="author-image img-fluid" src="@/assets/smiling-author.jpg" />
        </div>
        <div class="col-md-10">
          <p class="author-name text-truncate">
            By JohnDoe
          </p>
          <p class="message text-truncate">
            Greeting to the sun
          </p>
        </div>
      </div>
    </div>
  </main>
</template>

<script>
export default {
  
}
</script>

<style lang="scss" scoped>
@import './CreatorCard.scss';
</style>
