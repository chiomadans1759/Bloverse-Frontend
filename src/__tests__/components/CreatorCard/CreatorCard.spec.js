import { mount } from '@vue/test-utils'
import CreatorCard from '@/components/CreatorCard/CreatorCard.vue';

describe('CreatorCard Component', () => {
  it('should render correctly', () => {

    const wrapper = mount(CreatorCard , {});
    expect(wrapper.element).toMatchSnapshot();
  });
});
