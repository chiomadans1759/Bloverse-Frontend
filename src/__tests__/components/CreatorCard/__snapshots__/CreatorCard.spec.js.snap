// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreatorCard Component should render correctly 1`] = `
<main
  id="bl-author-card"
>
  <div
    class="d-none d-md-block d-lg-block"
  >
    <div
      class="row"
    >
      <div
        class="col-md-2 align-self-center p-0"
      >
        <img
          class="author-image img-fluid"
          src="@/assets/smiling-author.jpg"
        />
      </div>
       
      <div
        class="col-md-10"
      >
        <p
          class="author-name text-truncate"
        >
          
          By JohnDoe
        
        </p>
         
        <p
          class="message text-truncate"
        >
          
          Greeting to the sun
        
        </p>
      </div>
    </div>
  </div>
</main>
`;
